profile/README.md

```diff
@@ -0,0 +1,46 @@
+![Virtual Coffee full logo - brackets and carets making a mug shape before the words "Virtual Coffee"](../virtual-coffee-full.png)
+
+
+:coffee: Welcome to Virtual Coffee! :coffee:
+======
+[Virtual Coffee](https://virtualcoffee.io) is a laid-back conversation with developers twice a week. It's the conversation that keeps going in slack. It's the online events that support developers at all stages of the journey. It's the place you go to make friends. Anyone can join! Whether you're thinking about getting into tech or have been in it for decades.
+
+Our Mission
+------
+Virtual Coffee's mission is to be a welcoming tech community that allows room for growth and mentorship at all levels, and to create meaningful opportunities for learning, leadership, and contribution for everyone.
+
+[Sponsor Us!](https://github.com/sponsors/Virtual-Coffee)
+------
+Here are some of the things that sponsorship money will go to support:
+
+* Twice-Weekly Coffee Chats
+* Member-led Presentations and Workshops
```

Comment:
Let's add a link to the Lunch & Learn playlist: https://www.youtube.com/playlist?list=PLh9uT23TA65idCyc_orC85RefgY_-fKsG